language/src/main.rs

```diff
@@ -185,8 +195,36 @@ impl Callbacks for HacspecCallbacks {
                         .err("unknown backend extension for output file");
                     return Compilation::Stop;
                 }
+                };
+                
+                match &self.output_template_file {
+                    None => if self.copy_template {
+                        Command::new("cp")
+                            .arg(write_file.clone())
+                            .arg(write_file.clone() + "_template")
+                            .spawn()
+                            .expect("Failed copy to template")
+                    } else {
+                        return Compilation::Stop;
+                    },
+                    Some(template_file) => {
+                        Command::new("git")
```

Comment:
It's very likely that `git` is installed. But we should check if it's there before calling it or throw a comprehensive error here when it's not available.